src/plugins/azure/auth.ts

```diff
@@ -16,6 +16,11 @@ export const azLoginCommand = () => ({
   args: ["login"],
 });
 
+export const azLogoutCommand = () => ({
+  command: "az",
+  args: ["logout"],
+});
```

Comment:
Should the `az logout` command be added to the `azureSshProvider.reproCommands()` output for completeness?

Reply:
fixed 👍